index.html

```diff
@@ -0,0 +1,132 @@
+</head>
+
+<body>
+  <header>
+    <!-- video or image as a header is cool :) -->
+    <div id=hero>
+      <video autoplay loop muted preload="auto" id="graffitiVideo">
+        <source src="./videos/compressed_graffiti.mp4" type="video/mp4">
+        Your browser doesn't support HTML5 video.
+      </video>
+      <div class="content">
+        <h1>Graffiti artist bootcamp</h1>
+        <h2> Want to become a real graffiti artist ? Sign up to the graffit artist bootcamp this fall 2021.</h2>
+        <p><button id=signbutton class="w3-button w3-black w3-round-xxlarge">Sign up to Graffiti bootcamp</button></p>
+      </div>
+    </div>
+  </header>
+  <div class="content-container">
+    <!-- Signup form -->
+    <div class="form-group">
+      <form class="form" action="http://httpbin.org/anything" method="POST">
+        <div class="form-element">
+          <h2>Sign up to bootcamp</h2>
+        </div>
+        <div class="form-element">
+          <label for="fname">First name:</label><br>
+          <input name="firstname" id="fname" type="text" placeholder="firstname" required>
+        </div>
+        <div class="form-element">
+          <label for="lastname">Last name:</label><br>
+          <input name="lname" id="lname" type="text" placeholder="lastname" required>
+        </div>
+        <div class="form-element">
+          <label for="email">Email adress:</label><br>
+          <input name="email" id="email" type="email" placeholder="my@email.com">
+        </div>
+        <div class="form-element">
+          <label for="pasword">Password:</label><br>
+          <input name="password" id="password" type="password" placeholder="********">
+
+        </div>
+        <br>
+        <div class="form-element">
+          <textarea name="message">
+        Any other information.
+        </textarea>
+        </div>
+        <div class="form-element">
+          <label class="radio-text" name="radio">Should we send more info? </label><br>
+          <label for="radio">Rookie</label>
+          <input name="radio1" type="radio">
+          <label for="radio">Intermediate</label>
+          <input name="radio2" type="radio">
+          <label for="radio">Experienced</label>
+          <input name="radio3" type="radio">
+        </div>
+        <br>
+        <div class="form-element">
+          <label for="color">What is your favorit graffit color ? </label>
+          <br>
+          <select name="color">
+            <option value="pink">Pink</option>
+            <option value="blue">Blue</option>
+            <option value="green">Green</option>
+          </select>
+        </div>
+        <br>
+        <div class="form-element">
+          <label for="checkbox">Should we send more info? </label>
+          <input name="checkbox" type="checkbox">
+        </div>
+        <div class="form-element">
+          <input type="submit" value="Submit">
+        </div>
+      </form>
```

Comment:
Overall, this is a very clean, easy to read form structure with correct usage of labels and `form-element` classes as well as naming conventions.  Good job.